deliveries/societal-grant-2-milestone-1

```diff
@@ -0,0 +1,23 @@
+# Milestone Delivery :mailbox:
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**  
+
+
+
+* **Application Document:** [Societal Application](https://github.com/w3f/Grants-Program/blob/master/applications/societal_grant2.md) 
+* **Milestone Number:** 1
+
+
+
+## Deliverables
+
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 0a. | License |[GitHub repo link](https://github.com/sctllabs/societal-grant-2-submission/blob/main/LICENSE)| Apache 2.0 license included. |
+| 0b. | Documentation |[GitHub repo link](https://github.com/sctllabs/societal-grant-2-submission/blob/main/README.md) | Readme document contains instructions on how to run and deploy societal front-end. |
+| 0c. | Testing Guide |[GitHub repo link](https://github.com/sctllabs/societal-grant-2-submission/blob/main/docs/AssetsTestingGuide.md)| Guide on how to test Assets Pallet Lockable/Reservable traits using Polkadot-JS front-end. |
+| 0d. | Docker |[Docker Image](https://hub.docker.com/layers/societal/societal-node/vb4f81a4/images/sha256-69ba2805d6eb08d8bd5721afc4c82d2975a368efcde2634356ba6568a9111e9d?context=explore)| Dockerfile found at Societal's docker hub. The tag information is as follows: **docker pull societal/societal-node:vb4f81a4**. Please refer to the [Run in Docker](https://github.com/sctllabs/societal-grant-2-submission#run-in-docker) section of the README to run with the **vb4f81a4** tag specified. |
+| 0e. | Article |[Testing Article](https://github.com/sctllabs/societal-grant-2-submission/blob/main/docs/AssetsTestingGuide.md) |A tutorial that explains how to make use of Assets Pallet Lockable/Reservable traits using the basic Polkadot-JS UI. |
```

Comment:
Remove line. There is no article in the M1 delivery.

Reply:
done.